CMakeLists.txt

```diff
@@ -166,6 +166,7 @@ if(NOT INSTALLER_RUN)
 		endif()
 
 		find_package(Qt5Widgets ${FIND_MODE})
+		find_package(libcaffeine)
```

Comment:
Does it need to be found both here and in UI?

Reply:
I removed the redundant ones